packages/driver-adapter-utils/src/debug.ts

```diff
@@ -1,4 +0,0 @@
-import { Debug } from '@prisma/debug'
-
-export { Debug }
-export default Debug
```

Comment:
Wouldn't removing the default export constitute a slight breaking change? Are there driver adapters out there that use it maybe? If not, I think the change is ok.

Reply:
Code change reverted.